cmd/manager/main_test.go

```diff
@@ -8,14 +8,14 @@ import (
 
 	api "github.com/kubeflow/katib/pkg/api"
 	mockdb "github.com/kubeflow/katib/pkg/mock/db"
-	mockmodelstore "github.com/kubeflow/katib/pkg/mock/modelstore"
 )
 
 func TestCreateStudy(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 	mockDB := mockdb.NewMockVizierDBInterface(ctrl)
-	mockModelStore := mockmodelstore.NewMockModelStore(ctrl)
+	// Don't need test for Model Store. Right now Katib doesn't use ModelDB.
```

Comment:
Removing `mockModelStore` test is reasonable but it is not related to this PR. Could you split it to another PR?

Reply:
Should I split changing version of the cluster (https://github.com/kubeflow/katib/pull/328/files#diff-3d8f11bfc35c15745c970724619c57c3L36) in another PR, as well? 
Also, should I delete tests for mockModelStore or just comment it?

Reply:
>changing version of the cluster

Yes, please split it.

> `mockModelStore`

You can delete it.